Issue #10 Настроить BottomNavigationView

NekhoroshkinaElena opened the pull request:
Создала нижнюю панель навигации

NekhoroshkinaElena committed:
Добавила нижнюю панель навигации

NekhoroshkinaElena committed:
Исправление замечаний detect

SerjePerm commented on app/src/main/res/menu/bottom_navigation_menu.xml:
```diff
@@ -0,0 +1,17 @@
+    <item
+        android:id="@+id/searchFragment"
+        android:icon="@drawable/ic_bottom_search"
+        android:title="Главная" />
```
строковые ресурсы вынести в Strings.xml

SerjePerm commented on app/src/main/res/layout/activity_root.xml:
```diff
@@ -14,4 +14,14 @@
         app:defaultNavHost="true"
         app:navGraph="@navigation/navigation_graph" />
 
+    <com.google.android.material.bottomnavigation.BottomNavigationView
```
дизайн не соответствует макету: цвета, отступ между текстом и картинкой, нет разделительной полосы

SerjePerm commented on app/src/main/res/layout/activity_root.xml:
```diff
@@ -14,4 +14,14 @@
+        android:layout_height="56dp"
+        android:paddingBottom="8dp"
+        app:itemPaddingBottom="0dp"
+        app:itemPaddingTop="8dp"
```
дождаться файла dimens и отступы задать через res файл

NekhoroshkinaElena committed:
Merge branch 'refs/heads/dev' into feature/add-bottom-navigation

NekhoroshkinaElena committed:
fix: строковые ресурсы вынесены в String.xml
fix: нижняя панель навигации сделана в соответствии с макетом
fix отступы заданы через res файл

SerjePerm reviewed (commented):
all good
